app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -228,13 +239,15 @@ class InstalledAddonDetailsFragment : Fragment() {
                     runIfFragmentIsAttached {
                         this.addon = it
                         switch.isClickable = true
+                        enableSwitch.isClickable = true
                         view.remove_add_on.isEnabled = true
                     }
                 },
                 onError = {
                     switch.isChecked = addon.isAllowedInPrivateBrowsing()
```

Comment:
This line (not change here) should move into the `runIfFragmentAttached` block just in case.